aiohttp/websocket_client.py

```diff
@@ -33,65 +29,34 @@ class MsgType(IntEnum):
 closedMessage = Message(MsgType.closed, None, None)
 
 
-@asyncio.coroutine
-def ws_connect(url, protocols=(), timeout=10.0, connector=None,
-               response_class=None, autoclose=True, autoping=True, loop=None):
-    """Initiate websocket connection."""
+def ws_connect(url, *, protocols=(), timeout=10.0, connector=None,
+               ws_response_class=None, autoclose=True, autoping=True,
```

Comment:
I'm unsure about this. Would we still check that ```ws_response_class is None```? Otherwise, if the user were to call 

```python
ws_connect("ws...", response_class=None)
```

then *self._ws_response_class* on the ``ClientSession`` would be ``None``, and throw an error. Unless we change to ``ws_response_class=ClientWebSocketResponse``, and I still check in the function body that  ```ws_response_class is None```.